packages/storage/src/implementation/observer.ts

```diff
@@ -14,26 +14,66 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import * as type from './type';
+import { isFunction } from './type';
 import { FirebaseStorageError } from './error';
 
+/**
+ * @public
+ * Called once for each value in a stream of values.
```

Comment:
Suggestion: Function that is called once ...

Same for ErrorFn and CompleteFn below.

Reply:
Done.